.github/workflows/retention-maintenance.yml

```
@@ -0,0 +1,47 @@
+    - cron: "15 2 * * 0"
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  retention:
+    name: Weekly Retention Cleanup Drill
+    runs-on: ubuntu-latest
+    env:
+      CODEX_MULTI_AUTH_DIR: ${{ runner.temp }}/codex-retention-root
```

actionlint (1.7.11)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)